proto/misc.proto

```diff
@@ -72,8 +72,12 @@ message GetVersionResponse {
   string store = 1;
   // Big Query dataset
   string big_query = 2;
-  // Bigtable instance
-  string big_table = 3;
+  // Base Bigtable instance
```

Comment:
needs an aligned website change?

Reply:
Yes, website will use this API for versioning too.